<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textmemorise_createset</class>
 <widget class="QWidget" name="textmemorise_createset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main">
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create a set</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="setTitleBox">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name of the set:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="setTitle"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Contents of the set:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="setContents"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="knowGuage">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>How much of the set do you know?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="setKnowPercent">
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
